sdk/appconfiguration/app-configuration/src/appConfigurationClient.ts

```diff
@@ -109,11 +109,6 @@ export interface InternalAppConfigurationClientOptions extends AppConfigurationC
    * NOTE: this is an internal option, not for general client usage.
    */
   syncTokens?: SyncTokens;
-  /**
-   * Whether we want to run as if we're in node or in the browser.
-   * (currently only affects which name we use for the user agent header)
-   */
-  isNodeOverride?: boolean;
```

Comment:
I don't follow why we are removing this..

Reply:
The comment describes what it was for - we don't need to do this now that we can test the browser behavior in the actual browser.

Reply:
Ah, I thought it was used in the public API surface, nevermind